scripts/install/RegisterEvents.php

```diff
@@ -98,6 +100,31 @@ public function __invoke($params)
             ]
         );
 
+        // @todo This also needs a migration
+        $eventManager->attach(
+            ResourceUpdated::class,
+            [
+                ResourceUpdatedListener::class,
+                'listen'
+            ]
+        );
+        /*//initially the test is empty
```

Comment:
Importing a test triggers a `Resourceupdated` event, which in turn triggers the transformations to attach the `referenced_resources`.